packages/react/src/components/Menu/Menu.tsx

```diff
@@ -88,7 +88,7 @@ interface MenuProps extends React.HTMLAttributes<HTMLUListElement> {
   /**
    * Specify a DOM node where the Menu should be rendered in. Defaults to document.body.
    */
-  target?: Element;
+  menuTarget?: Element;
```

Comment:
I think it's appropriate for this to just be called `target` still. Renames are breaking changes, so we'd have to leave the old one in and map it to the new one, put a deprecation warning on the old one, and write a codemod so folks can easily migrate.

Reply:
I have added the target prop for backward compatibility issues. Added deprecation warning for target props.
Also added codemod for migration